crypto/fipsmodule/hmac/hmac.c

```diff
@@ -68,14 +68,19 @@
 uint8_t *HMAC(const EVP_MD *evp_md, const void *key, size_t key_len,
               const uint8_t *data, size_t data_len, uint8_t *out,
               unsigned int *out_len) {
+  FIPS_service_indicator_lock_state();
```

Comment:
It's good when you put a comment to explain why we decided to lock the indicator here as you did in many other places.